doc/man1/pkeyutl.pod

```diff
@@ -55,6 +57,19 @@ Print out a usage message.
 This specifies the input filename to read data from or standard input
 if this option is not specified.
 
+=item B<-rawin>
+
+This indicates that the input data is raw data, which is not hashed by any
+message digest algorithm. The user can specifies a digest algorithm by using
+the B<-digest> option.
+
+=item B<-digest algorithm>
```

Comment:
What happens if you don't specify this but you are using an algorithm that requires it?

Reply:
Depending on the public key algorithm - it can fail, generate a wrong signature or choose a default digest algorithm which finally gets the correct signature. Should this be also documented?

Reply:
I'm wondering whether pkeyutil should explicitly check this or not. AFAIK it's only Ed448/Ed25519 that don't require a digest. Or the other alternative would be to use a default (say SHA256) if one is not explicitly specified (which (I think??) would be ignored in Ed448/Ed25519).

Reply:
I'm not sure if we should do the explicitly check in pkeyutil app, it seems `EVP_DigestSignInit` has already done this check, so we could just simply let it do the job.

Reply:
It looks to me like it will use a sensible default. For RSA, DSA and ECDSA that seems to be SHA256. For EdDSA no digest is required. Not sure what happens for SM2. We should probably  document the default behaviour.

Reply:
Right, RSA/DSA/ECDSA uses SHA256 as the default digest algorithm. SM2 should use SM3 but current implementation has a problem - because SM2 shares the `ameth` with EC key so when someone wants to get the default digest algorithm he gets SHA256. This is fixed in commit 58d1fbf9, and I think this commit could be ported back to 1.1.1 as well.

For the document of `pkeyutl` I updated the doc for the default behavior if `-digest` is not present on the command line.